src/gestures/drag/utils/constraints.ts

```diff
@@ -71,7 +71,7 @@ export function calcRelativeAxisConstraints(
     }
 
     if (max !== undefined) {
-        constraints.max = Math.max(axis.min + max - length, axis.min + max)
+        constraints.max = Math.max(axis.min + max - length, axis.min)
```

Comment:
What does this change fix?

Reply:
This basically caps constraints to be a minimum of the size of the axis, whereas before it could get all junked up.